src/mbtest/server.py

```diff
@@ -152,6 +153,26 @@ def add_imposters(self, definition: Union[Imposter, Iterable[Imposter]]) -> None
             definition.attach(self.host, post.json()["port"], self.server_url)
             self._running_imposters.append(definition)
 
+    def add_stubs(self, definition: Union[Stub, Iterable[Stub]], port: int):
```

Comment:
A stub should be added to an imposter rather than a server, no?

Reply:
I followed this logic: 
One port - one imposter. If you know the port you can identify an imposter.
If an imposter exists - add a stub to this imposter. 
If there no imposter on the port, then create a new imposter and add your stub to it.

Reply:
Or you asked, why this method in Server class?
Yes, that makes the point. I will move it to Imposter

Reply:
Yup - I'd rather it was there. The impostor knows what its port is. And I'd rather adding a new impostor was explicit.

Reply:
Surry, I made a new pull-request by mistake https://github.com/brunns/mbtest/pull/36
Moved the method to Imposter.